client/extensions/woocommerce/state/action-types.js

```diff
@@ -1,3 +1,7 @@
+export const WOOCOMMERCE_API_SET_ERROR = 'WOOCOMMERCE_API_SET_ERROR';
+export const WOOCOMMERCE_API_CLEAR_ERROR = 'WOOCOMMERCE_API_CLEAR_ERROR';
+export const WOOCOMMERCE_API_CREATE_PRODUCT = 'WOOCOMMERCE_API_CREATE_PRODUCT';
```

Comment:
A small note: `WOOCOMMERCE_API_CREATE_PRODUCT_REQUEST` would conform better with the latest Calypso naming conventions.